packages/@coorpacademy-components/src/template/slides-review/index.js

```diff
@@ -0,0 +1,396 @@
+  context
+) => {
+  const {skin} = context;
+  const primarySkinColor = useMemo(() => getOr('#00B0FF', 'common.primary', skin), [skin]);
+
+  const slidesInitialState = useMemo(() => {
+    const states = new Map();
+    const {answer, question} = firstSlide;
+    // eslint-disable-next-line fp/no-loops
+    for (let index = 0; index < TOTAL_SLIDES_STACK; index++) {
+      const content = index === 0 ? {answer, question} : {};
+      states.set(index, {...content, hidden: false, position: index});
+    }
+    return states;
+  }, [firstSlide]);
+
+  const stepItemsInitialState = useMemo(() => {
+    const states = new Map();
+    // eslint-disable-next-line fp/no-loops
+    for (let index = 0; index < TOTAL_SLIDES_STACK; index++) {
+      const current = index === 0;
+      states.set(index, {current, value: `${index + 1}`, icon: null});
+    }
+    return states;
+  }, []);
+
+  const [finishedSlides, updateFinishedSlides] = useReducer(finishedSlidesReducer, new Map());
+
+  const [slidesState, updateSlides] = useReducer(slidePositionReducer, slidesInitialState);
+
+  const [stepItemsState, updateStepItems] = useReducer(stepItemsReducer, stepItemsInitialState);
+
+  const [revisionState, updateRevisionState] = useState('ongoing');
+
+  const builtStackedSlides = useMemo(() => {
+    const StackedSlides = [];
+    // eslint-disable-next-line fp/no-loops
+    for (let slideNumber = 0; slideNumber < TOTAL_SLIDES_STACK; slideNumber++) {
+      StackedSlides.push(
+        buildSlide(
+          slideNumber,
+          slidesState,
+          updateSlides,
+          primarySkinColor,
+          validate,
+          correctionPopinProps,
+          updateStepItems,
+          finishedSlides,
+          updateFinishedSlides,
+          updateRevisionState
+        )
+      );
+    }
+
+    return StackedSlides;
+  }, [slidesState, primarySkinColor, validate, correctionPopinProps, finishedSlides]);
+
+  const stepItemsArray = useMemo(() => {
+    // eslint-disable-next-line unicorn/prefer-spread
+    return Array.from(stepItemsState.values());
+  }, [stepItemsState]);
+
+  const _headerProps = {
+    ...headerProps,
+    steps: stepItemsArray,
+    key: 'review-header',
+    hiddenSteps: revisionState !== 'ongoing'
+  };
+
+  return (
+    <div
+      key="slides-revision-container"
+      data-name="slides-revision-container"
+      className={style.slidesRevisionContainer}
+    >
+      <div key="player-background-container" className={style.playerBackground}>
+        <ReviewBackground aria-label={reviewBackgroundAriaLabel} />
+      </div>
+
+      <ReviewHeader {..._headerProps} />
+
+      {revisionState === 'ongoing' ? (
+        <div
+          key="stacked-slides-container"
+          data-name="stacked-slides-container"
+          className={style.stackedSlidesContainer}
+        >
+          {builtStackedSlides}
+        </div>
+      ) : // slot toPlug congrats component
+      // <div className={style.congrats}/>
+      null}
```

Comment:
left commented intentionally, in order to flag where to plug the congrats component on the next iteration